### PR TITLE
Community Initiatives

### PR DESCRIPTION
**Pull request by Michae2xl**
Projects, Social Media, Sites, Network statistics, Explores, Aggregators by Decred Community

### CODE REVIEW

File: docs/Community Initiatives [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+Brazil: https://t.me/decredbr
+China: https://t.me/decred_cn
+Italia: https://t.me/DecredItalia
+
+YouTube 
+
+Brazil: https://www.youtube.com/channel/UC73wa2ddXuPWsmenVfeFTYg
+Decred Brazil Stake Pool: https://www.youtube.com/channel/UC1TdziC2MSZ8kU4gkt8KN_A
+Africa: https://www.youtube.com/channel/UCWj1aij3QmzTL4BDTyo0o4Q
+Decred Community: https://www.youtube.com/channel/UC8ozM0CknOMoQXrBgrIUS7A
```
Comment (dajohi) [inline docs/Community Initiatives]: https://www.youtube.com/decredchannel

Comment (Michae2xl) [reply docs/Community Initiatives]: Sure, Sure! Thank you

**Comment by jholdstock** [comment]
Hi @Michae2xl,

I'm not sure what the idea is here, but this PR will not be merged as it currently stands.

If the intention is to add a new page into docs.decred.org, then this will need to be added into `mkdocs.yml` and the new file would need to be formatted as a markdown document.

I think its worth having a conversation about if this something we would want to add into the docs site. Perhaps this could be discussed in a github issue, or in the documentation channel on Matrix.

**Comment by Michae2xl** [comment]
> Hi @Michae2xl,
> 
> I'm not sure what the idea is here, but this PR will not be merged as it currently stands.
> 
> If the intention is to add a new page into docs.decred.org, then this will need to be added into `mkdocs.yml` and the new file would need to be formatted as a markdown document.
> 
> I think its worth having a conversation about if this something we would want to add into the docs site. Perhaps this could be discussed in a github issue, or in the documentation channel on Matrix.


Hi @jholdstock 

Thank you for the feedback

Got it. Yep, basically I was thinking that could great have a space with all links to facilitate newcomers from different countries then they can find your media preference, language or which site to follow Decred.

**Comment by Michae2xl** [comment]
> Suggest to sort lists by alphabetical order ?

@Guang168 yes, absolutely. I'll do that, thanks for the suggestion

**Comment by xaur** [comment]
For those lurking, the list submitted by @Michae2xl in this PR was incorporated here: https://github.com/decredcommunity/wiki/blob/master/wiki/social-media.md

**Pull request closed without merging**